src/components/modules/DetailedPage/ProductHeader/StyledProductHeader.js

```diff
@@ -60,6 +60,12 @@ export const Username = styled.div`
   margin-right: 1rem;
   line-height: 18px;
 `;
+export const UsernameAvatar = styled.div`
```

Comment:
why are we displaying a div here? in the figma I see an avatar (img)

![image](https://user-images.githubusercontent.com/27841994/162551892-de831b05-965b-4b08-86e7-6ff245018e9f.png)